src/v1/controllers/EnrollWebauthnController.js

```diff
@@ -99,6 +99,10 @@ export default FormController.extend({
               return transaction.activate({
                 attestation: CryptoUtil.binToStr(newCredential.response.attestationObject),
                 clientData: CryptoUtil.binToStr(newCredential.response.clientDataJSON),
+                // example data: ["nfc", "usb"]
+                transports: JSON.stringify(newCredential.response.getTransports()),
```

Comment:
Will the backend care at all if `newCredential.response.getTransports()` is `null` or `undefined` (if it is even possible to have these values)?

I am just imagining some scenario, if we pass `newCredential.response.getTransports()` has the value `undefined`, we stringify it to `"undefined"`. Then in the backend, if we expect `null`, and it is instead the string `"undefined"` and it gets past a null check.